src/chain/response.rs

```diff
@@ -405,3 +405,56 @@ impl TryFrom<EventAttribute> for Tag {
         })
     }
 }
+
+impl From<tendermint_rpc::endpoint::abci_query::AbciQuery> for ChainResponse {
```

Comment:
This all looks good. 

Its a little frustrating that we have to re-implement these impls for the same type in rust (because they are re-exported from cosm-rs package vs using directly here): https://github.com/de-husk/cosm-tome/blob/38e54b01465056d60240aac3e53dfef5995c50dc/src/chain/response.rs#L40-L48